README.md

```diff
@@ -90,3 +90,15 @@ Instructions for generating $\Lambda$ gun events using pythia.
     $ vim myLambdaGun_cfi.py
 11) Generate a RAW .root file and a RECO .root file with 10 events. You'll may need to alter the options below to suit your needs.
     $ cmsDriver.py myLambdaGun_cfi.py -n 10 --mc --step GEN,SIM,DIGI,L1,DIGI2RAW,RAW2DIGI,RECO --conditions auto:mc --eventcontent RAWSIM,RECOSIM --datatier RAW,RECO
+    
+### Submitting crab jobs for MC generation
+12) Go to the dir with example config files. 
+   $ cd Configuration/Generator/python
+13) Run teh crab config file to submit 10 jobs with 100 events each, you can edit BOTH config files to adjust the number fo jobs and events:
```

Comment:
Maybe explain here that you specify the cfi file in crab_config.py?

Reply:
You mean in crab_MC1000.py?  And yes lines 46, 64 and 77 need be edited depending on the cfi file, I can specify that, but also the lambda gun specs (maxPt, particle ID..etc) can be edited in that file.